app/src/androidTest/java/com/github/fribourgsdp/radio/unit/PlayerGameHandlerTest.kt

```diff
@@ -15,9 +18,11 @@ import java.util.*
 
 
 class PlayerGameHandlerTest {
+    private val ctx: Context = ApplicationProvider.getApplicationContext()
+
     private lateinit var mockSnapshot: DocumentSnapshot
 
-    private val sleepingTime = 50L
+    private val sleepingTime = 500L
```

Comment:
Do you really need to increase the sleeping time?

Reply:
I think that even if it is for test of the same class, we can win some precious seconds by having the minimum time required to make the test run.

Reply:
So I put it back to 50L but I will see with @nathanduchesne why one of them was increased to 1000L.